dev-python/networkx/networkx-1.1.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="Python package for handling complex networks"
+DESCRIPTION="NetworkX is a Python package for the creation, manipulation, and \
+study of the structure, dynamics, and functions of complex networks."
+HOMEPAGE="https://www.networkx.github.io"
```

Comment:
```diff
-HOMEPAGE="https://www.networkx.github.io"
+HOMEPAGE="https://networkx.github.io/"
```
That's because the certificate is not valid for `networkx.github.io`, as we can see with e.g. curl:
```
$ curl -I https://www.networkx.github.io/ 
curl: (51) SSL: no alternative certificate subject name matches target host name 'www.networkx.github.io'

$ curl --silent -I https://networkx.github.io/  | head -1
HTTP/2 200 
```
(Also don't forget the trailing slash :)